src/routes/categories/components/__tests__/__snapshots__/CategoryList.spec.js.snap

```diff
@@ -19,15 +19,15 @@ exports[`CategoryList should render and display a caption 1`] = `
     <CategoryListItem
       category={
         CategoryModel {
-          "_availableLanguages": Object {},
-          "_content": "",
+          "_availableLanguages": undefined,
```

Comment:
Although, we don't actively write tests (so far) in this project, we should still make sure, that the existent ones at least make sense.
Please either delete the test, or adjust it so we don't have undefined values everywhere